sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v2/metadata.yaml

```diff
@@ -9,7 +9,7 @@ description: |-
 
   Fulfills https://mozilla-hub.atlassian.net/browse/ROAD-85
 owners:
-  - jklukas@mozilla.com
+  - ctroy@mozilla.com
```

Comment:
Happy to overtake this. For my edification, under what circumstances are we using v2 of this, versus when should we use v3?

Reply:
- And I assume v1 is deprecated/not in use?

Reply:
The v1 query did nothing more than remove city-level geo data from the impressions. It has been removed.

v2 can go away once we are sufficiently comfortable with v3 as production-grade solution and we have adapted any downstream queries to use v3.